### PR TITLE
Tallying

### PR DESCRIPTION
**Pull request by benknoble**

### CODE REVIEW

File: input/labor-anonymized.md [hunk 1]
```diff
@@ -33,9 +34,9 @@
 * A theoretical connection between Qi and Arrows in category theory was drawn.
 * The idea of a Qi-themed event was suggested
 * The Qi design challenge was organized
-* An illustrative example to motivate design improvements to the `feedback` form of the language was provided
-* The design of `feedback` was improved
-* The PR improving the design of `feedback` was reviewed
```
Comment (countvajhula) [inline input/labor-anonymized.md:38]: I think in this particular case, the original ones here reflected a later stage of discussion. I can revert it after merging though.

Comment (benknoble) [reply input/labor-anonymized.md:38]: Ah, you mean it should have been changes in the evaluation trees rather than the inputs?

Comment (countvajhula) [reply input/labor-anonymized.md:38]: No, just that the versions that were replaced here were what we had originally changed the text _to_. I restored them in [this commit](https://github.com/countvajhula/dia-qi/commit/98b2c49b9b27e33c9f21511c306e7611c92e3b99). Lmk if that doesn't look right to you.

Comment (countvajhula) [reply input/labor-anonymized.md:38]: The reason we made the change IIRC is that "example implementation" suggests that it is an implementation of the feature, and thus a different level of attribution than "illustrative example to motivate" the implementation, which is what it actually was in this case.

**Pull request merged**